src/resolvers/__tests__/user.spec.ts

```diff
@@ -37,6 +37,7 @@ describe('User queries', () => {
     expect(res.body.singleResult.data?.users).toBeDefined();
 
     const result = res.body.singleResult.data?.users as [User];
+    // console.log(result);
```

Comment:
L40: Did you mean to leave this commented out "console.log()" line?

Reply:
yeah, that test is currently failing so I need to get back to it